lib/steps.js

```diff
@@ -43,15 +43,17 @@ to detect that.
 regex thoughts: https://stackoverflow.com/questions/171480/regex-grabbing-values-between-quotation-marks
 */
 
-setDefaultTimeout( 120 * 1000 );
+// Hoping that most load and download times will be under 30 seconds.
+// TODO: Check what average load and download times are in our current interviews.
+setDefaultTimeout( 30 * 1000 );
```

Comment:
At some point, it would be nice to encode the `30` into a variable (something like `DEFAULT_GLOBAL_TIMEOUT`) so we don't have to repeat it in multiple places.

Reply:
#322. I thought I planned to do that at some point and then thought there was a barrier. Something about not being able to get external context values into cucumber functions. I now think I hallucinated that, so we should definitely try again.